packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -185,20 +185,34 @@ export const layerHostStyle: IStyle = {
 /**
  * @private
  */
-export const localVideoCameraCycleButtonStyles: IButtonStyles = {
-  root: {
-    position: 'absolute',
-    right: '0.1rem',
-    top: '0.1rem',
-    color: '#FFFFFF', // only shows up on running video feed to we want to force specific colours.
-    zIndex: 2, // shows the button directly over the local video feed.
-    background: 'transparent'
-  },
-  rootFocused: {
-    // styles to remove the unwanted white highlight and blue colour after tapping on button.
-    color: '#FFFFFF',
-    background: 'transparent'
-  }
+export const localVideoCameraCycleButtonStyles = (theme: Theme): IButtonStyles => {
+  return {
+    root: {
+      position: 'absolute',
+      display: 'flex',
+      width: _pxToRem(36),
+      height: _pxToRem(32),
+      flexDirection: 'row',
+      left: _pxToRem(32),
```

Comment:
Not understanding these positioning styles -- why `left` over `right` here? And what is flex&flexDirection needed for?
Consider wrapping the icon in a stack and stack item having the stack item have `align="end"` to avoid css

Reply:
I can switch it but I used styles from the Figma to match the design.